manifests/webhook.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: admission-webhook-example-svc
+  namespace: pingcap
```

Comment:
Leave the namespace field to allow user specify it from the command line.

Reply:
Oh, seems `namespace` is required by cluster role binding. So you can keep it here, but please change it to `tidb-admin` the default namespace in our documentations.